### DISCUSSION TITLE
PNG to PDF is creating an invalid file

### DISCUSSION THREAD

**Discussion by gitexplr** [dlemstra/Magick.NET#745] [Help] [answered]
I am trying to save the file as PDF, it is creating an invalid file, can some one help?

`ImageMagick.MagickImage im = new ImageMagick.MagickImage("testFile1.png");
im.Write("testFile1.pdf");`

**Comment by gitexplr** [comment]
Can you please let me know if this is not MagickNET issue?

**Reply by dlemstra** [reply]
I have not had time to research this. Hopefully this weekend.

**Comment by t246246** [comment, marked as answer]
How invalid?
I can create pdf from your file using just installed from nuget Q16-AnyCPU v7.22.0.  Project is set to use .NET Framework 4.8.
Generated pdf can be seen FoxIt reader and Adobe Acrobat Reader.  pdfinfo reports no problem.

**Reply by gitexplr** [reply]
thank you so much, I haven't tried with Q16-AnyCPU v7.22.0 and NET Framework 4.8. It is working fine.
